go.mod

```diff
@@ -1,14 +1,18 @@
 module gopkg.in/DataDog/dd-trace-go.v1
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.5
```

Comment:
I believe adding toolchain directives in our repo is one of the things we agreed to avoid
```suggestion
```

Reply:
Oops, not sure what tool or command I ran that added these, I'll remove them

Reply:
I've wondered the same thing. It happens when the `go get`/`go mod` command updates the `go` directive due some dependency requiring the newer release. If you manually change the `go` directive yourself before doing any dependency changes, the `toolchain` directive won't be added. Ref: https://go.dev/ref/mod#go-mod-file-toolchain